pkg/convert/pot.go

```diff
@@ -121,36 +177,116 @@ func writeToPoTag(b *bytes.Buffer, tag poTag, str string) error {
-		lines[len(lines)-1] = lines[len(lines)-1] + "\\n" // add the "\n" back to the last line
+		lines[len(lines)-1] += "\\n" // add the "\n" back to the last line
 	}
 
-	if len(lines) == 1 {
-		_, err = fmt.Fprintf(b, "%s \"%s\"\n", tag, lines[0])
-		if err != nil {
-			return fmt.Errorf("write %s: %w", tag, err)
+	return lines, nil
+}
+
+func extractPluralForms(plural string) ([]string, error) {
+	if plural == "" {
+		return nil, fmt.Errorf("plural is empty")
+	}
+
+	plural = strings.ReplaceAll(plural, "{$count}", "%d")
+	// Use regular expressions to extract the plural forms from the plural finds string in {}
+	re := regexp.MustCompile(`when [\d*]+ \{([^}]+)\}`)
+	matches := re.FindAllStringSubmatch(plural, -1)
+
+	if len(matches) == 0 {
+		return nil, fmt.Errorf("no plural forms found in plural")
+	}
+
+	// Create a slice of strings to hold the extracted plural forms
+	pluralForms := make([]string, 0, len(matches))
+
+	// Loop through the matches and extract the plural form from each one
+	for _, match := range matches {
+		pluralForms = append(pluralForms, match[1])
+	}
+
+	return pluralForms, nil
+}
+
+func convertToFormatMessage(plurals []string) string {
+	var sb strings.Builder
+
+	sb.WriteString("match {$count :number}\n")
+
+	for i, plural := range plurals {
+		line := strings.ReplaceAll(strings.TrimSpace(plural), "%d", "{$count}")
+
+		var count string
+
+		switch i {
+		default:
+			count = fmt.Sprintf("%d", i+1)
+		case len(plurals) - 1:
+			count = "*"
 		}
 
-		return nil
+		sb.WriteString(fmt.Sprintf("when %s {%s}\n", count, line))
 	}
 
-	_, err = fmt.Fprintf(b, "%s \"\"\n", tag)
-	if err != nil {
-		return fmt.Errorf("write %s: %w", tag, err)
+	return sb.String()
+}
+
+func writeMessage(b *bytes.Buffer, index int, message model.Message) error {
+	if index > 0 {
+		if _, err := fmt.Fprint(b, "\n"); err != nil {
+			return fmt.Errorf("write new line: %w", err)
+		}
 	}
 
-	for _, line := range lines {
-		if strings.HasSuffix(line, "\\n") {
-			_, err = fmt.Fprint(b, "\""+line+"\""+"\n")
-			if err != nil {
-				return fmt.Errorf("write %s: %w", tag, err)
-			}
-		} else {
-			_, err = fmt.Fprint(b, "\""+line+"\\n\""+"\n")
-			if err != nil {
-				return fmt.Errorf("write %s: %w", tag, err)
+	if message.PluralID != "" {
+		count := strings.Count(message.Message, "when")
+		if _, err := fmt.Fprintf(b, "\"Plural-Forms: nplurals=%d; plural=(n != 1);\\n\"\n", count); err != nil {
+			return fmt.Errorf("write plural forms: %w", err)
+		}
+	}
+
+	descriptions := strings.Split(message.Description, "\n")
+
+	for _, description := range descriptions {
+		if description != "" {
+			if _, err := fmt.Fprintf(b, "#. %s\n", description); err != nil {
+				return fmt.Errorf("write description: %w", err)
 			}
 		}
 	}
 
+	if message.Fuzzy {
+		if _, err := fmt.Fprint(b, "#, fuzzy\n"); err != nil {
+			return fmt.Errorf("write fuzzy: %w", err)
+		}
+	}
+
+	if err := writeTags(b, message); err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
```suggestion
	return writeTags(b, message)
```

Reply:
try to be consistent with error messages, use Present Simple everywhere in errors
e.g
```go
fmt.Errorf("divide ...")
fmt.Errorf("convert ...")
```